Source/WebCore/platform/ios/VideoFullscreenInterfaceAVKit.mm

```diff
@@ -54,6 +54,10 @@
 #import <AVKit/AVPictureInPictureController.h>
 #endif
 
+#if HAVE(UIKIT_WEBKIT_INTERNALS)
+#include <WebKitAdditions/VideoFullscreenInterfaceAVKitAdditions.h>
```

Comment:
(Nit - `#import` for consistency with the other header includes?)

Reply:
Thanks for the review!